src/Symfony/Component/Form/Tests/Extension/Core/Type/UrlTypeTest.php

```diff
@@ -86,6 +93,7 @@ public function testThrowExceptionIfDefaultProtocolIsInvalid()
     public function testSubmitNullUsesDefaultEmptyData($emptyData = 'empty', $expectedData = 'http://empty')
     {
         $form = $this->factory->create(static::TESTED_TYPE, null, [
+            'default_protocol' => 'http', // TODO: remove in 7.0
```

Comment:
that's fine if it stays as is so better remove all similar todo noticed. Less merge conflicts ahead also.
```suggestion
            'default_protocol' => 'http',
```